test/packages/rust/src/generated/instructions/add_config_lines.rs

```diff
@@ -0,0 +1,194 @@
+        pub fn invoke(&self) -> solana_program::entrypoint::ProgramResult {
+            self.invoke_signed(&[])
+        }
+        #[allow(clippy::vec_init_then_push)]
+        pub fn invoke_signed(
+            &self,
+            signers_seeds: &[&[&[u8]]],
+        ) -> solana_program::entrypoint::ProgramResult {
+            let instruction = solana_program::instruction::Instruction {
+                program_id: crate::MPL_CANDY_MACHINE_CORE_ID,
+                accounts: vec![
+                    solana_program::instruction::AccountMeta::new(*self.candy_machine.key, false),
+                    solana_program::instruction::AccountMeta::new_readonly(
+                        *self.authority.key,
+                        true,
+                    ),
+                ],
+                data: self.args.try_to_vec().unwrap(),
+            };
+            let mut account_infos = Vec::with_capacity(2 + 1);
+            account_infos.push(self.program.clone());
+            account_infos.push(self.candy_machine.clone());
+            account_infos.push(self.authority.clone());
+
+            if signers_seeds.is_empty() {
+                solana_program::program::invoke(&instruction, &account_infos)
+            } else {
+                solana_program::program::invoke_signed(&instruction, &account_infos, signers_seeds)
+            }
+        }
+    }
+
+    /// `add_config_lines` CPI instruction builder.
+    pub struct AddConfigLinesBuilder<'a> {
+        program: &'a solana_program::account_info::AccountInfo<'a>,
+        candy_machine: Option<&'a solana_program::account_info::AccountInfo<'a>>,
+        authority: Option<&'a solana_program::account_info::AccountInfo<'a>>,
+        index: Option<u32>,
+        config_lines: Option<Vec<ConfigLine>>,
+    }
+
+    impl<'a> AddConfigLinesBuilder<'a> {
+        pub fn new(program: &'a solana_program::account_info::AccountInfo<'a>) -> Self {
+            Self {
+                program,
+                candy_machine: None,
+                authority: None,
+                index: None,
+                config_lines: None,
+            }
+        }
+        pub fn candy_machine(
+            &'a mut self,
+            candy_machine: &'a solana_program::account_info::AccountInfo<'a>,
+        ) -> &mut Self {
+            self.candy_machine = Some(candy_machine);
+            self
+        }
+        pub fn authority(
+            &'a mut self,
+            authority: &'a solana_program::account_info::AccountInfo<'a>,
+        ) -> &mut Self {
+            self.authority = Some(authority);
+            self
+        }
+        pub fn index(&'a mut self, index: u32) -> &mut Self {
+            self.index = Some(index);
+            self
+        }
+        pub fn config_lines(&'a mut self, config_lines: Vec<ConfigLine>) -> &mut Self {
+            self.config_lines = Some(config_lines);
+            self
+        }
+        pub fn build(&'a self) -> AddConfigLines {
```

Comment:
```suggestion
        pub fn build(&'a self) -> AddConfigLines<'a> {
```
missing lifetime specifier